contracts/Pausable.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity >=0.4.21 <0.6.0;
+
+contract Pausable {
+  address public owner;
+  bool public paused;
```

Comment:
So you are inviting the developer of the child contracts to change the values at will?